Luleherll/turing-backend-challenge
# #168504832 User should be able to signup

### What does this PR do?
- Enables a customer to create an account

### How to manually test it
- Start the server and make a `POST` request to `http://localhost:80/customers` with the required payload:
```
{
"name": string,
"email": string,
"password": string
}
```
and you should be able to see this response:
```
{
    "customer": {
         "customer_id" : integer,
         "name" : string,
         "email" : string,
         "address_1" : null,
         "address_2" : null,
         "city" : null,
         "region" : null,
         "postal_code" : null,
         "shipping_region_id" : integer,
         "credit_card" : null,
         "day_phone" : null,
         "eve_phone" : null,
         "mob_phone" : null,
     },
    "accessToken": string,
    "expiresIn": string,
}

```

### How has it been tested?
- Integration testing

### PT stories
#168504832
